src/mock/handlers/auth.ts

```diff
@@ -89,4 +91,26 @@ export default [
         // deletes the cookie
         return res(ctx.cookie('jwt', ''), ctx.status(200));
     }),
+
+    rest.put('/api/users/me/reset-password', (req, res, ctx) => {
+        return res(ctx.status(200));
+    }),
+    rest.put('/api/users/me/name', (req, res, ctx) => {
+        const { fname, lname } = req.body as { fname: string, lname: string }
+        let toReturnUser = makeUser();
+        toReturnUser.name.first = fname;
+        toReturnUser.name.last = lname;
+        return res(
+            ctx.status(200),
+            ctx.json({
+                user: toReturnUser
+            })
+        );
+    }),
+    rest.put('/api/users/email', (req, res, ctx) => {
+        const { email } = req.body as { email: string };
+        let toReturnUser = makeUser();
+        toReturnUser.email.address = email
+        return res(ctx.status(200), ctx.json({ user: toReturnUser }));
+    }),
```

Comment:
One thing I would recommend is just putting in a simple failure case for your endpoints, meaning `ctx.status(4xx)` (probably 400), so you can easily test the UI in case of failure.  Ex. make the request fail if the email submitted is `fail@fail.com` or if the first and last name is `fail`.

Reply:
Done, fail cases added.